ajax/libs/taggle/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "taggle",
+  "filename": "taggle.min.js",
+  "version": "1.7.1",
+  "description": "Taggle is a dependency-less tagging library",
+  "author": "Sean Coker <sean@seancoker.com>",
+  "url": "http://sean.is/poppin/tags",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:okcoker/taggle.js.git"
+  },
+  "homepage": "https://sean.is/poppin/tags",
+  "keywords": [
+    "tags",
+    "tagging",
+    "standalone",
+    "micro",
+    "library"
+  ],
+  "license": "MIT",
+  "npmName": "taggle",
+  "npmFileMap": [
+    {
+      "basePath": "example/js",
```

Comment:
Maybe it is ok. I'm not sure.

```json
"basePath": "example",
"files": [
    "js/taggle*" 
] 
```

```json
"basePath": "",
"files": [
    "example/js/taggle*" 
] 
```
 Let's ask @PeterDaveHello which one is better.